integration-tests/src/main/java/org/apache/polaris/service/it/test/PolarisApplicationIntegrationTest.java

```diff
@@ -93,6 +93,18 @@
 import org.junit.jupiter.api.TestInfo;
 import org.junit.jupiter.api.extension.ExtendWith;
 
+/**
```

Comment:
Previously with dropwizard we could set overrides that were test-specific; is that not possible now?